kivy/uix/actionbar.py

```diff
@@ -249,6 +249,10 @@ class ActionGroup(ActionItem, Spinner):
        defaults to 'normal'.
     '''
 
+    dropdown_width = NumericProperty(0)
+    '''If non zero, provides the width for the associated DropDown.
```

Comment:
I agree.  I even wanted to add a test and that's when I discovered that the graphics tests are messed up https://github.com/kivy/kivy/issues/3130